File: Java/LT01_ESTSEQ/LT01_ESTSEQ_11.java
/*
 * Nome Do Programador: Lucas Sartorelli
 * Data: 13/02/2025
 * Descrição: Receba o raio de uma circunferência. 
 * Calcule e mostre o comprimento da circunferência.
 */
import javax.swing.JOptionPane;
public class LT01_ESTSEQ_11 {
	public static void main (String args[]){
		Double raio;
		raio = Double.parseDouble(JOptionPane.showInputDialog("Insira o vaor do raio: "));
		raio *= (2*Math.PI);
		JOptionPane.showMessageDialog(null, "O Valor da Circunferência é " +raio);
	}
}

File: C/lt_01_estseq/lt01_estseq_11.c
// Nome do Progrmador:
// Descrição:

#include <math.h>
#include <stdio.h>

int main() {

  double raio, area;

  printf("Insira o valor do Raio: \n");
  scanf("%lf", &raio);

  area = raio * M_PI * M_PI;
  raio = raio * 2 * M_PI;

  printf("O perimetro do cirulo será %.2lf.\nE a area será %.2lf.\n", raio,
         area);
  return 0;
}
